dresscode/uvotimsum.py

```diff
@@ -259,10 +259,8 @@ def calc_coicorr_uncertainty(
     primary_counts_sum_hdul = fits.open(primary_counts_sum_fname)
     # this is in counts but we need to convert to fraction by dividing by corrected summed counts (primary)
 
-    new_data = coicorr_unc_sq_hdul[1].data / primary_counts_sum_hdul[1].data
-
-    # todo: After summing, take the square root
-    # new_data = np.sqrt(coicorr_unc_sq_hdul[1].data) / primary_counts_sum_hdul[1].data
+    # we summed squares, square root the summed uncertainties
+    new_data = np.sqrt(coicorr_unc_sq_hdul[1].data) / primary_counts_sum_hdul[1].data

```

Comment:
@mdecleir similar question, but we want to take the square root of the summed (and squared) uncertainty counts _first_ and _then_ divide by the primary counts to go back to uncertainties, right?

Reply:
I am not sure why we are doing this here.
The result of this division gives you the relative uncertainty (a percentage). I believe, before, we were saving the uncertainty in counts/second, not the relative uncertainty. With before, I mean in the original pipeline.

Reply:
Correction, I think this is ok. The conversion to Janskys is done in the calibration.py if I am not mistaken. Can you please confirm @falkben?

Reply:
correct, conversion to janskys is done in `calibration.py` here: https://github.com/spacetelescope/DRESSCode/blob/main/dresscode/calibration.py#L23

Reply:
it seems like we don't need that fraction and could skip the division and then multiplication, by the primary data.

After taking the sqrt of the summed, squared uncertainties in counts, we are already in janskys, right? if so, i can make that change to the code. it shouldn't change anything, and won't make anything noticeably faster, but it does make things a little less confusing

Reply:
Is the primary data the same in the division and multiplication (i.e., are no additional steps happening in between?). If they are the same, you could probably skip that step, but make sure the header is still correct.

You would be saving the uncertainties in counts, which is not the same as Janskys.

Reply:
ah, i see -- i think the primary gets converted to janskys, so it would not be the same. in that case i think we need to keep things as they are

Reply:
Exactly! That's what I thought, but I wasn't sure. Best to keep things as they are indeed.

Reply:
Also, please make sure the header keyword is correct, to avoid any confusion.